platform/linux-generic/include/odp_packet_io_internal.h

```diff
@@ -231,6 +235,18 @@ static inline void pktio_cls_enabled_set(pktio_entry_t *entry, int ena)
 	entry->s.enabled.cls = !!ena;
 }
 
+static inline int _odp_pktio_tx_ts_enabled(pktio_entry_t *entry)
```

Comment:
Is it ok to have a leading underscore in the name?

Reply:
The project convention is to use `_odp` -prefix for implemention internal functions.